api/URL.json

```diff
@@ -150,9 +150,15 @@
             "ie": {
               "version_added": false
             },
-            "nodejs": {
-              "version_added": "20.0.0"
-            },
+            "nodejs": [
+              {
+                "version_added": "19.9.0"
+              },
+              {
+                "version_added": "18.17.0",
+                "version_removed": "19.0.0"
```

Comment:
Where is information from that it was removed in 19.0.0?

Reply:
It's because the feature was added in v18.17.0 and v19.9.0!  Therefore, it was not available in v19.0.0 to v19.8.0.

Reply:
Thank you! Got it now.